src/nspawn/nspawn-mount.c

```diff
@@ -185,9 +187,11 @@ static int tmpfs_patch_options(
                 const char *selinux_apifs_context,
                 char **ret) {
 
-        char *buf = NULL;
+        char *buf = strdup(options);
+	if (!buf)
```

Comment:
hmm, this doesn't look right btw. The next if block might override buf, and leak its memory that way?